aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -211,8 +210,7 @@ pub struct AptosVM {
     pub(crate) gas_feature_version: u64,
     gas_params: Result<AptosGasParameters, String>,
     pub(crate) storage_gas_params: Result<StorageGasParameters, String>,
-    /// For a new chain, or even mainnet, the VK might not necessarily be set.
-    pvk: Option<PreparedVerifyingKey<Bn254>>,
+    keyless_config: KeylessConfig,
```

Comment:
What is the lifetime of this config? @zjma @alinush? Per epoch, per block?

Reply:
It is up to us but here we implemented it as per-block.